src/containers/Cfe/style.css

```diff
@@ -24,6 +24,7 @@
     flex: 2;
     background-color: black;
     max-height: calc(100vh - var(--header-height) - var(--tab-bar-height));
+    padding-right: 120px;
 }
```

Comment:
Style the viewer and plots so they have the same padding (same width as the thumbnail gallery)